aws/b2b_personalization/_env/alb.hcl

```
@@ -9,10 +9,19 @@ terraform {
 
 dependency "vpc" {
   config_path = "../vpc"
+
+  mock_outputs = {
+    vpc_id             = "vpc-dummy-id"
+    subnet_private_ids = ["10.0.0.0/24", "10.0.0.1/24"]
+  }
 }
 
 dependency "sgs" {
   config_path = "../sgs"
+
+  mock_outputs = {
+    security_group_id = "sg-dummy-id"
```

```suggestion
    security_group_id = "sg-dummy-id"
    mock_outputs_allowed_terraform_commands = ["validate", "providers"]
```
Is this not missing here?